boilerplate/package.json.ejs

```diff
@@ -8,9 +8,9 @@
     "format:js": "prettier --write {.,**}/*.js",
     "format:json": "prettier --write {.,**}/*.json",
     "format:md": "prettier --write {.,**}/*.md",
-    "format:ts": "prettier --write {.,**}/*.{ts,tsx} && tslint --fix -p .",
+    "format:ts": "prettier --write {.,**}/*.{ts,tsx}",
```

Comment:
At this point Prettier should be handling all code formatting so no need to kick off a ESLint fix here I don't think.